.env.django

```diff
@@ -70,3 +70,7 @@ EMAIL_FILE_PATH='/app/mail'
 USE_STREAMING=False
 COMPRESSION_ENABLED=False
 FILE_EXPIRY_IN_DAYS=30
+
+# === Playwright Tests ===
+
+USER_EMAIL=
```

Comment:
It can't be left blank, but adding a default won't work as it needs to be whatever user you have setup. I've taken a couple of extra steps now:
* Make the instruction in the README clearer
* Throw an error as soon as possible if `USER_EMAIL` not set with a clear explanation of how to fix